packages/xstate-wallet/src/utils/json-rpc-utils.ts

```diff
@@ -1,35 +1,41 @@
-import {Outcome, isAllocationOutcome, convertBytes32ToAddress} from '@statechannels/nitro-protocol';
+import {
+  Outcome,
+  isAllocationOutcome,
+  convertBytes32ToAddress,
+  State,
+  convertAddressToBytes32
+} from '@statechannels/nitro-protocol';
 import {utils} from 'ethers';
-import {NETWORK_ID, CHALLENGE_DURATION} from '../constants';
+import {NETWORK_ID, CHALLENGE_DURATION, ETH_ASSET_HOLDER_ADDRESS} from '../constants';
 import {
   UpdateChannelParams,
   Allocations,
   CreateChannelParams
 } from '@statechannels/client-api-schema/types';
-import {Channel, State} from '@statechannels/wallet-protocols';
+import {Channel} from '@statechannels/wallet-protocols';
 
-// function createAllocationOutcomeFromParams(params: Allocations): Outcome {
-//   return params.map(p => {
-//     return {
-//       // TODO: Need to look up the the asset holder for the token
-//       assetHolderAddress: ETH_ASSET_HOLDER_ADDRESS,
-//       allocation: p.allocationItems.map(a => {
-//         return {
-//           destination: convertAddressToBytes32(a.destination),
-//           amount: a.amount
-//         };
-//       })
-//     };
-//   });
-// }
+function createAllocationOutcomeFromParams(params: Allocations): Outcome {
+  return params.map(p => {
+    return {
+      // TODO: Need to look up the the asset holder for the token
+      assetHolderAddress: ETH_ASSET_HOLDER_ADDRESS,
+      allocation: p.allocationItems.map(a => {
+        return {
+          destination: convertAddressToBytes32(a.destination),
+          amount: a.amount
+        };
+      })
+    };
+  });
+}
 
 export function createJsonRpcAllocationsFromOutcome(outcome: Outcome): Allocations {
   return outcome.map(o => {
     if (!isAllocationOutcome(o)) {
       throw new Error('Attempted to convert non allocation outcome to an allocation');
     }
     return {
-      token: o.assetHolderAddress,
+      token: '0x0',
```

Comment:
Use `AddressZero`?

Reply:
Right now the API specifies that `token:0x0` is what the API will return for ETH so I'm going to leave as is for now.